CONTRIBUTING.md

```diff
@@ -5,6 +5,7 @@
 - Deno >= v1.9.2
 - [Vercel](https://vercel.com/)
 - GitHub API v4
+- Docker and Docker compose (opcional)
```

Comment:
optional in English? opcional is Portuguese？

Reply:
Good catch, changed.